templates/rhdm73-authoring-ha.yaml

```diff
@@ -724,7 +840,7 @@ objects:
               command:
               - "/bin/bash"
               - "-c"
-              - "curl --fail --silent -u '${KIE_ADMIN_USER}:${KIE_ADMIN_PWD}' http://localhost:8080/kie-drools-wb.jsp"
```

Comment:
I think RHDM uses kie-drools-wb.jsp whereas RHPAM uses kie-wb.jsp. Please verify.

Reply:
@errantepiphany Actually, for 7.3 the Workbenches were unified so just kie-wb.jsp is available for both.

Reply:
@sutaakar Nice! Thank you. :)